docs/content/Contribution guidelines/operations/packaging-and-versioning.md

```diff
@@ -15,19 +15,14 @@ archives, the Kubernetes API group, and the space framework component.
 
 KubeStellar defines a Kubernetes [API
 group](https://kubernetes.io/docs/reference/using-api/#api-groups),
-named `edge.kubestellar.io`. API groups are versioned; see [the
+named `control.kubestellar.io`. API groups are versioned; see [the
```

Comment:
This whole document is obsolete.  The maintained content is `docs/content/v0.20/packaging.md` .